AsyncDisplayKit/ASCellNode.h

```diff
@@ -93,6 +93,29 @@ typedef NSUInteger ASCellNodeAnimation;
  */
 - (void)setNeedsLayout;
 
+/**
+ * @abstract Initializes a cell with a given viewControllerBlock.
+ *
+ * @param viewControllerBlock The block that will be used to create the view controller whose root view will be used to create the backing view.
+ *
+ * @return An ASCellNode created using the root view of the view controller provided by the viewControllerBlock.
+ * The view controller's root view is resized to match the calcuated size produced during layout.
+ *
+ */
+- (instancetype)initWithViewControllerBlock:(ASDisplayNodeViewControllerBlock)viewControllerBlock;
```

Comment:
Just to reduce API surface area, I think we should probably kill this variant.  Passing nil as the didLoadBlock is pretty reasonable especially since this will not be a very common call in people's apps.